data/en/array.json

```diff
@@ -4,13 +4,28 @@
 	"syntax": "array()",
 	"returns": "array",
 	"related": ["arrayNew"],
-	"description": "Creates a new array populated with any arguments that have been passed to the function.",
+	"description": "Creates a new array populated with any arguments that have been passed to the function. It is recommend by lucee to use the inline array notation.",
```

Comment:
I see you added `It is recommend by lucee to use the inline array notation.` so are you saying that they discourage using this function? If so you can add a key for this, eg:

`"discouraged": "Lucee recommends using the inline array notation instead of the array function. See example below for details",`